docs/onboard.md

```diff
@@ -2,26 +2,26 @@
 
 ## Public Officer
 ### Step 1
-Send in your request for a TechPass account to TechPass Team
-
-Once your request has been approved, you will receive an email invitation.
-Click on accept invitation link
-![PO_Invite](assets/support/PO_Invite.jpeg)
-![PO_Invite1](assets/support/PO_Invite1.jpeg)
-
+On techpass homepage, click Sign Up & sign up with your official work email
```

Comment:
proposed change:
Click Sign Up on TechPass home and sign up with your official WoG email

add a note, use ?>
If you are a SE-GSIB user, please raise a request to TechPass to create an account [here] (https://go.gov.sg/techpass-sr)

Reply:
ah sorry forgot about SE-GSIB user. updated, thanks!